### PR TITLE
Fix nightly 28/01/2021

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add an ID for the save button on Add new query form
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      |  TEST_PATH="functional/BO/14_advancedParameters/07_database/sqlManager/01_CRUDSqlQuery.js" URL_FO=yourShopURL npm run specific-test
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23037)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/RequestSql/Blocks/form.html.twig [hunk 1]
```diff
@@ -60,7 +60,7 @@
         <a href="{{ path('admin_sql_requests_index') }}" class="btn btn-outline-secondary">
           {{ 'Cancel'|trans({}, 'Admin.Actions') }}
         </a>
-        <button class="btn btn-primary float-right">
+        <button id="save_btn" class="btn btn-primary float-right">
```
Comment (NeOMakinG) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/RequestSql/Blocks/form.html.twig]:
```suggestion
        <button id="sql-query-save-btn" class="btn btn-primary float-right">
```

File: tests/UI/pages/BO/advancedParameters/database/sqlManager/add.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ class AddSQLQuery extends BOBasePage {
     // Selectors
     this.sqlQueryNameInput = '#sql_request_name';
     this.sqlQueryTextArea = '#sql_request_sql';
-    this.saveButton = '#main-div button.btn-primary';
+    this.saveButton = '#save_btn';
```
Comment (NeOMakinG) [inline tests/UI/pages/BO/advancedParameters/database/sqlManager/add.js]:
```suggestion
    this.saveButton = '#sql-query-save-btn';
```

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/RequestSql/Blocks/form.html.twig (hunk 1 above)
Comment (nesrineabdmouleh) [reply src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/RequestSql/Blocks/form.html.twig]: Done thx :+1:

File: tests/UI/pages/BO/advancedParameters/database/sqlManager/add.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/advancedParameters/database/sqlManager/add.js]: Done thx :+1:

**Review by NeOMakinG — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**